content/docs/dispensary/prescriptions.md

```diff
@@ -425,6 +425,61 @@ Once the prescription has been confirmed:
 - You can no longer edit prescription lines
 - You can no longer delete the prescription
 
+## Printing Labels
+
+Open mSupply has basic label printing that will be built upon with more flexibility in the future.
+
+![Prescription label](/docs/dispensary/images/prescription_label.jpg)
+![Prescription abbreviation](/docs/dispensary/images/prescription_abbreviation.png)
+
+First ensure that you have configured a printer in [Settings](/docs/settings/devices/). Note that at the moment we use the ZPL printing language under the hood, which covers about half the hardware on the market. You'll need to tweak the printer configuration (directly on the device) to get it just right for our label printing.
+
+### Directions
+
+For each item in the prescription you'll need to define item directions to be printed on the labels. These can be entered manually, with abbreviations or default item directions.
+
+The fastest way to do entry is to have abbreviations configured that allow fast entry of common directions. These are [configured on the mSupply Primary Server](https://docs.msupply.org.nz/dispensing:abbreviations). With some abbreviations configured, you should be able to enter them in the first text field "Abbreviation" as below. Once focus is changed it will translate the abbreviation to the expanded text in the "Directions" box. Further modifications may be made from there.
```

Comment:
"Once focus is changed" -> "Once you click outside the input field" perhaps? Slightly less tech-y?

Reply:
Thanks! Yea it was a bit techy better than onBlur though 😆